### PR TITLE
Home task:HTML 01

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mskokova/01_HTML/task_02.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>HOW-TO: Программа на Python</TITLE>
+	</HEAD> 
+	<BODY> 
+		<H1>HOW-TO: Программа на Python, Часть 3</H1>
+		<EM><UL>
```
Comment (pluhin) [inline mskokova/01_HTML/task_02.html]: tag em should be in each li tag
see on slides, how to formatting inline tags

File: mskokova/01_HTML/task_02.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			<LI>Программа на Python — часть 1</LI>
+			<LI>Программа на Python — часть 2</LI>
+			<LI>Программа на Python — часть 3</LI>
+		</UL></EM>
+		<H2>Содержание</H2>
+		<H3>HOW-TO: Программа на Python, Часть 3</H3>
+		<INS><UL>
```
Comment (pluhin) [inline mskokova/01_HTML/task_02.html]: inline tags should be inside blocks tags

File: mskokova/01_HTML/task_01.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	<BODY> 
+		<H1>HOW-TO: Программа на Python, Часть 2</H1>
+		<EM><UL>
+			<LI>Программа на Python — часть 1</LI>
+			<LI>Программа на Python — часть 2</LI>
+			<LI>Программа на Python — часть 3</LI>
+		</UL></EM>
+		<H2>Содержание</H2>
+		<H3>HOW-TO: Программа на Python, Часть 2</H3>
+		<INS><UL>
```
Comment (pluhin) [inline mskokova/01_HTML/task_01.html]: inline tags should be inside blocks tags

File: mskokova/01_HTML/task_01.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>HOW-TO: Программа на Python</TITLE>
+	</HEAD> 
+	<BODY> 
+		<H1>HOW-TO: Программа на Python, Часть 2</H1>
+		<EM><UL>
```
Comment (pluhin) [inline mskokova/01_HTML/task_01.html]: inline tags should be inside blocks tags

**Review by pluhin — CHANGES_REQUESTED** [review]

File: mskokova/01_HTML/task_02.html (hunk 1 above)
Comment (ghost) [reply mskokova/01_HTML/task_02.html]: Got it. Thanks. 
Should i correkt the mistakes and send homework again?

Comment (pluhin) [reply mskokova/01_HTML/task_02.html]: > Got it. Thanks.
> Should i correkt the mistakes and send homework again?

need to do changes in same files and commit them, i will see this correction

File: mskokova/01_HTML/add_task.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+		<PRE>
+ x | o |
+-----------
+   | х |
+-----------
+ o | x | o
+		</PRE>
+		<HR>
+		<H2>Тег address</H2>
+		<p><EM>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</EM></p>
```
Comment (pluhin) [inline mskokova/01_HTML/add_task.html]: And where is tag address?

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Any news here?

**Comment by ghost** [comment]
No news is good news.
I got confused in the github, but already figured it out (I hope).

File: mskokova/01_HTML/add_task.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+		обед с 13<sup><INS>00</INS></sup> до 14<sup><INS>00</INS></sup><BR>
+		выходной: <INS>суббота<INS>, <INS>воскресенье<INS></p>
+		<HR>
+		<H2>Вот такой текст</H2>
+		<H2><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> 
+		<sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> 
+		<sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub> <sup>т</sup></H2>
+		<HR>
+		<H2>Акроним и курсив</H2>
+		<p>ТФ: Y(<i>x</i>)=e<sup><i>­iπx</i></sup>+log<sub><i>x</i></sub>4</p>
```
Comment (pluhin) [inline mskokova/01_HTML/add_task.html:69]: Could you please replace  pi here to mnemonic

**Review by pluhin — APPROVED** [review]

**Pull request merged**